source/ui.html.md

```diff
@@ -13,20 +13,20 @@ Kubeless as a Web based UI available at [https://github.com/kubeless/kubeless-ui
 
 If you want to run the UI locally, check the [README](https://github.com/kubeless/kubeless-ui/blob/master/README.md), otherwise you can launch the UI directly within your Kubernetes cluster using a manifest like so:
 
-```
+~~~
 kubectl create -f https://raw.githubusercontent.com/kubeless/kubeless-ui/master/k8s.yaml
-```
+~~~
 
 The service is a NodePort for straightforward testing on minikube.
 
 ## Access the UI
 
 On minikube this will be:
 
-```
+~~~
 minikube service ui -n kubeless
-```
+~~~
 
 And you will then access the UI below:
 
-<img src="assets/images/kubeless-ui.png">
+<img src="/assets/images/kubeless-ui.png">
```

Comment:
Same here:

![Alt text for the image. It's very important for accessibility](/assets/images/kubeless-ui.png)